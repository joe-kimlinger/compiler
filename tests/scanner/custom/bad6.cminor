1231234^;
(-1232)
123\3
