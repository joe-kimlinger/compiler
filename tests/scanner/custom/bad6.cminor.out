INTEGER_LITERAL 1231234
EXPONENT
SEMICOLON
LPAREN
MINUS
INTEGER_LITERAL 1232
RPAREN
INTEGER_LITERAL 123
